src/borg/testsuite/archiver/argparsing.py

```diff
@@ -11,6 +12,23 @@ def test_bad_filters(self):
         self.cmd(f"--repo={self.repository_location}", "create", "test", "input")
         self.cmd(f"--repo={self.repository_location}", "delete", "--first", "1", "--last", "1", fork=True, exit_code=2)
 
+    def test_highlander(self):
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        self.cmd(f"--repo={self.repository_location}", "create", "--comment", "comment 1", "test-1", src_dir)
+        error_msg = "There can be only one"
+        # Default permission is 077
+        # Test that it works with a oen time specified default or custom value
+        output_default = self.cmd(f"--repo={self.repository_location}", "--umask", "077", "rlist")
+        assert error_msg not in output_default
+        output_custom = self.cmd(f"--repo={self.repository_location}", "--umask", "007", "rlist")
+        assert error_msg not in output_custom
+        # Test that all combinations of custom and default values fail
+        for first, second in combinations_with_replacement(["007", "077"], 2):
```

Comment:
guess in such an easy case we rather don't want to use `itertools. combinations_with_replacement`, but just give the 2 tuples directly.